### DISCUSSION TITLE
Farbik Website issue

### DISCUSSION THREAD

**Discussion by btesec** [trobfab/fabrik#619] [General]
Hello All Devs, notice the Fabrik website has been infected by malicious code

![infecte2](https://github.com/trobfab/fabrik/assets/33397670/00b059de-8b72-40bd-95eb-b483e177a050)
![infected](https://github.com/trobfab/fabrik/assets/33397670/acc42b27-beb1-443c-b5f5-41e890dd7717)

**Comment by trob** [comment]
Thanks for reporting, we are investigating.

**Comment by trob** [comment]
Site is restored, hopefully fixed.

**Comment by btesec** [comment]
Noted with thanks.

On Wed, Dec 6, 2023 at 2:58 PM troester ***@***.***> wrote:

> Site is restored, hopefully fixed.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/trobfab/fabrik/discussions/619#discussioncomment-7781331>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AH6ZXJSJYDEBBOTOHDR6TJ3YIDL7BAVCNFSM6AAAAABAJW4Z76VHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM3TOOBRGMZTC>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
Let's keep In Touch! - ***@***.*** - ***@***.***
***@***.*** - ***@***.***
